setup.py

```diff
@@ -11,7 +11,7 @@
 
 setup(
     name='remodel',
-    version='0.4.4',
+    version='0.4.5',
```

Comment:
I usually bump versions after merging pull requests and before drafting a release. Many times there are more pull requests that form a release, so that's my reasoning.